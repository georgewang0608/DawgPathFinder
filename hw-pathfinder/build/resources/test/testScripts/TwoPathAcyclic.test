# Find a path with Dijkstra's algorithm when
# there exist two paths between the nodes in an acyclic graph

CreateGraph simpleGraph

AddNode simpleGraph A
AddNode simpleGraph B
AddNode simpleGraph C
AddNode simpleGraph D

AddEdge simpleGraph A B 10.0
AddEdge simpleGraph A C 10
AddEdge simpleGraph C D 2
AddEdge simpleGraph B D 1

FindPath simpleGraph A D