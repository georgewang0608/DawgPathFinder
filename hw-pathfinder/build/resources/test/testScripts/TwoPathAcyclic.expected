# Find a path with Dijkstra's algorithm when
# there exist two paths between the nodes in an acyclic graph

created graph simpleGraph

added node A to simpleGraph
added node B to simpleGraph
added node C to simpleGraph
added node D to simpleGraph

added edge 10.000 from A to B in simpleGraph
added edge 10.000 from A to C in simpleGraph
added edge 2.000 from C to D in simpleGraph
added edge 1.000 from B to D in simpleGraph

path from A to D:
A to B with weight 10.000
B to D with weight 1.000
total cost: 11.000